packages/dropbox/package.js

```diff
@@ -0,0 +1,29 @@
+Package.describe({
+  name: 'cfs:dropbox',
+  version: '0.0.1',
+  summary: 'Dropbox storage adapter for CollectionFS',
+  git: 'https://github.com/CollectionFS/Meteor-cfs-dropbox.git',
```

Comment:
@raix I think just leave as the .git URL and use the `readme` field to set a specific read me. Does that work?